avm/res/network/network-manager/connectivity-configuration/main.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_generator": {
+      "name": "bicep",
+      "version": "0.25.53.49325",
```

Comment:
Placeholder to merge after publish `--with-source` is enabled

Reply:
I guess those and other comments should now be resolved and the PRs merged? Cc @eriqua